### ISSUE TITLE
no such file or directory zappajs/vendor/jquery-1.8.3.min.js

### ISSUE THREAD

**Issue by jbilcke** [zappajs/zappajs#56]
Hello, 

I've just installed the default (stable?) zappajs module, globally (also tested locally, same results),
using npm install -g zappajs

seems it worked:

```
zappajs@0.4.13 /usr/local/lib/node_modules/zappajs
├── node-uuid@1.4.0
├── uglify-js@1.3.4
├── coffee-css@0.0.5 (underscore@1.4.3, coffee-script@1.4.0)
├── coffeecup@0.3.17 (optparse@1.0.3, coffee-script@1.3.3, uglify-js@1.2.6, stylus@0.27.2)
├── express@3.0.0 (methods@0.0.1, fresh@0.1.0, range-parser@0.0.4, cookie@0.0.4, crc@0.2.0, commander@0.6.1, debug@0.7.0, mkdirp@0.3.3, send@0.1.0, connect@2.6.0)
└── socket.io@0.9.10 (policyfile@0.0.4, redis@0.7.2, socket.io-client@0.9.10)
```

however when I try to execute this sample:

``` CoffeeScript
require('zappajs') ->
  @get '/hi': 'hi'
```

I get:

```
Error: ENOENT, no such file or directory '/usr/local/lib/node_modules/zappajs/vendor/jquery-1.8.3.min.js'
    at Object.fs.openSync (fs.js:338:18)
    at Object.fs.readFileSync (fs.js:182:15)
    at vendor (/usr/local/lib/node_modules/zappajs/lib/zappa.js:22:15)
    at Object.<anonymous> (/usr/local/lib/node_modules/zappajs/lib/zappa.js:25:12)
    at Object.<anonymous> (/usr/local/lib/node_modules/zappajs/lib/zappa.js:1044:4)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:362:17)
```

I looked at this /vendor dir, and indeed there are only:

```
     sammy-0.7.2.min.js 
     socket.io.min.js
```

no jquery. however I don't know which side is right, the code or the folder :)

**Comment by shimaore** [comment]
The coder is definitely wrong in both cases.
Looks like a mistake in publishing 0.4.13, I'll correct this shortly, sorry for the problem.

**Comment by shimaore** [comment]
I just published 0.4.14 which has the proper jQuery in it.

**Comment by jbilcke** [comment]
thank you for the quick fix!
